docker-compose.yml

```diff
@@ -2,7 +2,7 @@
 services:
   # run postgres for proxy service to store observability metrics
   postgres:
-    image: postgres:15
+    image: postgres:13.12
```

Comment:
Why we need to downgrade?

Reply:
It's not downgrade, it was just to keep the version same with prod (on prod we had Postgres 13.12). I don't see any reason why we should keep different versions